man/systemd.unit.xml

```diff
@@ -1374,11 +1374,23 @@
 
           <para>If the <varname>systemd.condition-needs-update=</varname> option is specified on the kernel
           command line (taking a boolean), it will override the result of this condition check, taking
-          precedence over any file modification time checks. If it is used
+          precedence over any file modification time checks. If the kernel command line option is used,
           <filename>systemd-update-done.service</filename> will not have immediate effect on any following
           <varname>ConditionNeedsUpdate=</varname> checks, until the system is rebooted where the kernel
           command line option is not specified anymore.</para>
-          </listitem>
+
+          <para>Note that to make this scheme effective, the timestamp of <filename>/usr/</filename> should
+          be explicitly updated after its contents are modified. The kernel will automatically update
+          modification timestamp on a directory only when immediate children of a directory are modified; an
+          update which only updates nested files will not automatically result in mtime of
+          <filename>/usr/</filename> being updated.</para>
+
+          <para>Also note that if the update method includes a call to execute appropriate post-update steps
+          itself, it should <emphasis>not</emphasis> touch the timestamp of <filename>/usr/</filename>. In a
```

Comment:
i don't think the emphasis is necessary. I mean, it's OK if people do this, just not as efficient. The emphasis indicates that this was a really big problem I think (at least that's how I'd read it), but it's not, it's just a matter of doing things the most efficient way. Hence, I'd just write "not" here, without the `<emphasis>`.